src/microbe_stage/organelle_unlocks/UnlockProgress.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Stores which organelles have been unlocked by the player.
+/// </summary>
+[UseThriveSerializer]
+public class UnlockProgress
+{
+    [JsonProperty]
+    public bool UnlockAll;
+
+    /// <summary>
+    ///   The organelles that the player can use.
```

Comment:
I'd add "The default value is the by default unlocked organelles (ones without a condition)" here as it took me a bit to understand that key part here.